roles/load-balance/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Import global variables
+  include_vars:
+    dir: "{{ inventory_dir }}/group_vars/"
+
+- name: Check for nginx
+  package:
+    name: nginx
+    state: present
```

Comment:
Since this is covered in the `nginx` role, I think it is acceptable to drop this task here. It should be expected the `nginx` role is already applied in order for the `load-balancer` role to be useful. This is my personal preference, but it's up to you to keep or drop it.